--- a/CMakeLists.txt.orig	2017-05-24 06:42:48.560608961 -0600
+++ b/CMakeLists.txt	2017-05-24 06:43:06.711079176 -0600
@@ -69,7 +69,7 @@
   # Now check if we can use visibility to selectively export symbols
   # Get the GCC version - from KDE4 cmake files
   exec_program(${CMAKE_C_COMPILER} ARGS --version OUTPUT_VARIABLE	_gcc_version_info)
-  string (REGEX MATCH "[345]\\.[0-9]\\.[0-9]" _gcc_version "${_gcc_version_info}")
+  string (REGEX MATCH "[3456]\\.[0-9]\\.[0-9]" _gcc_version "${_gcc_version_info}")
   # gcc on mac just reports: "gcc (GCC) 3.3 20030304 ..." without the
   # patch level, handle this here:
   if (NOT _gcc_version)
